public/less/d2h.less

```diff
@@ -1,218 +1,434 @@
 /*
-*
-* Diff to HTML (diff2html.css)
-* Author: rtfpessoa
-* Modified by: codingtwinky
-* Date: Friday 29 August 2014
-* Last Update: Saturday 27 September 2014
-*
-*/
-
-.d2h-wrapper {
-  display: block;
-  margin: 0 auto;
-  text-align: left;
-  width: 100%;
-}
+ *
+ * Diff to HTML (diff2html.css)
+ * Author: rtfpessoa
+ *
```

Comment:
Maybe add the version number of diff2html here.

Reply:
Not sure, this part is copied 1:1

Reply:
Opted to include the css file from npm package so this file now only contains the adjustments.